charts/kyuubi/templates/kyuubi-alert.yaml

```diff
@@ -20,9 +20,7 @@ apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
   name: {{ .Release.Name }}
-  labels:
-    {{- include "kyuubi.labels" . | nindent 4 }}
+  labels: {{- include "kyuubi.labels" . | nindent 4 }}
```

Comment:
well, but the previous one is more intuitive for me, since the rendered result of template
```
  labels:
    {{- include "kyuubi.labels" . | nindent 4 }}
```
is something like
```
  labels:
    label1: value1
    label2: value2
```

any specify reason to use the proposed one?

Reply:
There is no specific reason, rather than following one style across all templates. The idea is to keep templates more compact if it is possible. We use this formatting style in many places, for instance: https://github.com/apache/kyuubi/blob/0a53415d929dfe477f9c31f707e559c09dd175a0/charts/kyuubi/templates/kyuubi-role.yaml#L25 https://github.com/apache/kyuubi/blob/0a53415d929dfe477f9c31f707e559c09dd175a0/charts/kyuubi/templates/kyuubi-service.yaml#L27 https://github.com/apache/kyuubi/blob/0a53415d929dfe477f9c31f707e559c09dd175a0/charts/kyuubi/templates/kyuubi-service.yaml#L44

Reply:
I agree to unify the style, but lean to invert the style.

Reply:
There is no strong convention at the moment as far as I know, so both approaches are valid. Personally I prefer more compact style, because from my experience it's more convenient to read and review the code, especially if a template has many lines. One argument here might be that Bitnami uses this approach as well for all their [charts](https://github.com/bitnami/charts/tree/main/bitnami).